### DISCUSSION TITLE
Welcome to tqec Discussions!

### DISCUSSION THREAD

**Discussion by vtomole** [tqec/tqec#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by afowler** [comment]
Austin Fowler: here to teach TQEC so we can all build tools to study this field. Ask me anything!

**Comment by johnzl-777** [comment]
Hope I'm not too late here! John Long, Scientific Software Developer at QuEra Computing. 

My work mostly centers around more "mechanical" things than what folks might think of in traditional quantum software like supporting multithreading for differential equation solvers, improving documentation on our software development kit, and figuring out how to make it as easy as possible for folks to develop things for quantum hardware.

I have two interests/goals in the community, the first being the opportunity to learn more about QEC and the second being the opportunity to contribute to open source tools that can help make quantum computing more accessible.

**Comment by doczenwiry** [comment]
Greetings everyone, Seweryn from Belgium here,

I'm interested to contribute to this project. I have been programming in Python since 2010, although not regularly and have an interest in the theoretical aspects of QC. Any suggestions for a specific issue I could look into are welcome.

A bit about my background perhaps. I got my PhD in (classical) computing science in 2015 and have been giving university lectures on a variety of topics since 2018 (operating systems, programming, program proving, algorithms, realtime computing). Over the last few years, I've been spending time studying QC on the side (I attach some fancy notes I wrote down on selected topics; I'm quite happy with the diagrams for the QFT and the order-finding application of the QPE). I feel like I want to stop just learning about it and contribute with a goal to transition to a job in the field (preferably research).

[dzw3-quantum-computing.pdf](https://github.com/user-attachments/files/24100128/dzw3-quantum-computing.pdf)

**Comment by afowler** [comment]
Always best to start with the github issues and reach out to someone
associated with one that appeals to you. Also look forward to seeing you at
group meetings!
